assets/scss/settings/vendor/sweetalert2/_settings.scss

```diff
@@ -0,0 +1,54 @@
+// SweetAlert2
+// -----------------------------------------------------------------------------
+
+
+//
+// Purpose: Settings for the vendor popup box replacement, SweetAlert2
+//
+// -----------------------------------------------------------------------------
+
+// Font color, size, family
+$swal-font-body:                                      stencilFontFamily("body-font"), Arial, Helvetica, sans-serif;
```

Comment:
All these variables can probably be generalised as well. i.e.: `$alert-font-body`